packages/compiler/test/config/config.test.ts

```diff
@@ -1,19 +1,25 @@
-import { deepStrictEqual, strictEqual } from "assert";
-import { dirname, join, resolve } from "path";
+import { deepStrictEqual, fail, strictEqual } from "assert";
+import { dirname, join, normalize, resolve } from "path";
```

Comment:
Should try not to use anything from `path` anywhere but scripts. This cause issues with windows vs unix path and everything in the compiler should already have been normalized to unix paths. The compiler provide some path functions that can be used instead.